src/docs/cookbook/networking/web-sockets.md

```diff
@@ -36,9 +38,9 @@ from the server and push messages to the server.
 In Flutter, use the following line to
 create a `WebSocketChannel` that connects to a server:
 
-<!-- skip -->
+<?code-excerpt "lib/main.dart (connect)" replace="/_channel/channel/g"?>
 ```dart
-final channel = IOWebSocketChannel.connect('ws://echo.websocket.org');
+final channel = IOWebSocketChannel.connect('wss://echo.websocket.org');
```

Comment:
Not sure about the difference here, but the complete example was using `wss`

Reply:
`wss` is the tls encrypted version of `ws`.